packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,289 @@ components:
             created
           example: 1587667174725
 
+    asset:
+      type: object
+      table: asset
+      additionalProperties: false
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        type:
+          type: string
+          enum:
+            - video
+            - audio
+          description: Type of the asset.
+          example: video
+        userId:
+          type: string
+          readOnly: true
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
```

Comment:
So I found that this field is considered "private" on the streams, which is likely because most of the time it will be referencing an object that cannot be retrieved by the user (an object store in Ivan's account). I believe we should follow the same pattern here, and make this field a `writeOnly` property, which will never be returned by the API **unless** an admin is making the request (so that `task-runner` can access it).